README.md

```diff
@@ -1 +1,47 @@
-# tf_ansible
\ No newline at end of file
+# tf_ansible
+
+## Module usage
+
+### Add special section to the playbook
+You must add this section on the top of playbook that will be used for provision
+
+e.g. `../ansible/playbooks/playbook.yml`
+* Create a runtime inventorty with an ip address of a host
+* Waite for target host is ready for ssh connection
+
+```
+---
+- hosts: localhost
+  vars:
+    gather_facts: true
+  tasks:
+  - name: Add public ip addresses to an dynamic inventory
+    add_host:
+      name: "{{ host }}"
+      groups: all
+
+  - local_action: wait_for port=22 host="{{ host }}" search_regex=OpenSSH delay=10
```

Comment:
@osterman `wait_for port=22` option allows to pause ansible execution for 300s (by-default) until 22 port becomes accessible.